src/commands/scaling/editScalingRange.ts

```diff
@@ -56,6 +56,8 @@ async function validateInput(range: string | undefined): Promise<string | undefi
         return localize('maxRangeExceeded', 'The maximum number of replicas is 10.');
     } else if (min > max) {
         return localize('minHigherThanMax', 'The minimum range cannot be larger than the maximum range.');
+    } else if (max === 0) {
```

Comment:
Should we try to handle negative numbers or rely on the `formatRegex` to take care of that?

Reply:
The format regex should take care of it since it has to start with a digit and there can only be one "-" before the next digit.

Side node: Why does node.js use RegExp as shorthand for regular expression when the rest of the world uses regex........